package.json

```diff
@@ -113,13 +113,10 @@
     "redux-api-middleware": "^3.0.1",
     "redux-thunk": "^2.3.0",
     "select2": "^4.0.6-rc.1",
-    "swagger-ui": "3.25.0",
+    "swagger-ui": "^3.51.1",
```

Comment:
I would stick to using exact versions to prevent more unexpected errors. Not with swagger-ui specifically but with any NPM package.
@didierofrivia @damianpm opinions?

Reply:
Not sure, maybe having this unexpected errors is good, because it notifies that something has changed

Reply:
@didierofrivia, WDYT ?

Reply:
Let's merge this, and later we decide if we should block all dependencies to exact versions, or not